launch_frontend/launch_frontend/action_parse_methods.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for launch action parsing methods."""
+
+import shlex
+
+import launch
+from launch.conditions import IfCondition
+from launch.conditions import UnlessCondition
+from launch.substitutions import TextSubstitution
+
+from .entity import Entity
+from .expose import expose_action
+from .parser import Parser
+
+
+def parse_action(entity: Entity, parser: Parser):
+    """
+    Parse action.
+
+    This is only intented for code reusage, and it's not exposed.
```

Comment:
@ivanpauno consider:
```suggestion
    This is only intended for code reuse, and it's not exposed.
```